db/migrate/20180807211758_create_legal_aid_applications.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLegalAidApplications < ActiveRecord::Migration[5.2]
+  def change
+    create_table :legal_aid_applications do |t|
+      t.string :application_ref
+
+      t.timestamps
```

Comment:
Just a note for anyone else puzzled by this. It's an ActiveRecord macro that results in the `created_at` and `updated_at` columns.

Reply:
yep rails gives you these audit columns as part of its conventions